packages/flutter_test/lib/src/binding.dart

```diff
@@ -375,6 +387,7 @@ abstract class TestWidgetsFlutterBinding extends BindingBase
         );
       } else {
         _pendingExceptionDetails = details;
+        reportExceptionNoticed(details); // mostly this is just a hook for the LiveTestWidgetsFlutterBinding
```

Comment:
Not sure if it matters, but the docs say that `reportExceptionNoticed ` is called "when there is no pending exception". So perhaps this line should run before we set `_pendingExceptionDetails`?

Reply:
Yeah, that's reasonable. Will switch that around.